packages/manager/modules/anthos/src/dashboard/routing.js

```diff
@@ -3,15 +3,72 @@ export default /* @ngInject */ ($stateProvider) => {
     url: '/:serviceName',
     component: 'anthosDashboard',
     resolve: {
+      breadcrumb: /* @ngInject */ (serviceName) => serviceName,
+
+      serviceName: /* @ngInject */ ($transition$) =>
+        $transition$.params().serviceName,
+
+      tenant: /* @ngInject */ (serviceName, AnthosTenantsService) => {
+        return AnthosTenantsService.getTenantDetails(serviceName);
+      },
+
+      hosts: /* @ngInject */ (serviceName, AnthosTenantsService) =>
+        AnthosTenantsService.getHosts(serviceName).then(({ data }) => data),
+
+      netappStorage: /* @ngInject */ (serviceName, AnthosTenantsService) =>
+        AnthosTenantsService.getTenantStorageInfo(serviceName),
+
+      publicIPs: /* @ngInject */ (serviceName, AnthosTenantsService) =>
+        AnthosTenantsService.getTenantPublicIPs(serviceName).catch(() => []),
+
+      privateIPs: /* @ngInject */ (serviceName, AnthosTenantsService) =>
+        AnthosTenantsService.getTenantPrivateIPs(serviceName).catch(() => []),
```

Comment:
Isnt `error` handled on routing directly ? Why the empty catch block here ?

Reply:
Because, for API produce an issue.
To avoid blocking dev, i returned an empty array.
API are aware and the issue will be fixed.

Reply:
We will have to track this and remove it once the API is fixed

Reply:
I added a TODO comment for that.